lldb/source/Commands/CommandObjectThread.cpp

```diff
@@ -1239,11 +1242,12 @@ class CommandObjectThreadSelect : public CommandObjectParsed {
       new_thread = process->GetThreadList().FindThreadByIndexID(index_id).get();
       if (new_thread == nullptr) {
         result.AppendErrorWithFormat("Invalid thread #%s.\n",
```

Comment:
Do we want to add some extra text saying "Invalid thread index #%s" to make it clear the thread index is invalid? Since we mention the "Invalid thread ID %lu" below?